opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -53,6 +54,18 @@ namespace Opm {
         const auto& cartDims = Opm::UgGridHelpers::cartDims(grid);
         setupCartesianToCompressed_(Opm::UgGridHelpers::globalCell(grid),
                                     cartDims[0]*cartDims[1]*cartDims[2]);
+
+        is_shut_or_defunct = [&ebosSimulator](const Well& well) {
+                if (well.getStatus() == Well::Status::SHUT)
+                    return true;
+                if(ebosSimulator.gridView().comm().size()==1)
```

Comment:
Nitpicks: space after if, spaces around `==`, braces.

Reply:
You forgot the one missing after the `if` :speak_no_evil: